clients/pkg/promtail/targets/file/tailer.go

```diff
@@ -149,15 +166,27 @@ func (t *tailer) readLines() {
 			continue
 		}
 
+		var text string
+		if t.decoder != nil {
+			var err error
+			text, err = t.convertToUTF8(line.Text)
+			if err != nil {
+				level.Error(t.logger).Log("msg", "failed to convert encoding", "error", err)
```

Comment:
Since we now send the error message in the log line itself, and we have a metric when this happens, I think this should be debug level, it feels mostly just likely to be noise to me at error level.